lib/src/io_utils.dart

```diff
@@ -91,7 +91,11 @@ class SubprocessLauncher {
   }
 
   SubprocessLauncher(this.context, [Map<String, String> environment]) {
-    if (environment == null) this._environment = new Map();
+    if (environment == null) {
```

Comment:
Could put this in the ctor as 

```dart
SubprocessLauncher(this.context, [Map<String, String> environment]) :
  this._environment = environment ?? <String, String>{};
```

Then `_environment` could be final?

Reply:
done